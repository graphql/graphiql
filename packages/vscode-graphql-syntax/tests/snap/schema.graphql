type Query {
  field: String
}

type Mutation {
  doTheThing(input: InputType!): String
}

input InputType {
  field: String
}
