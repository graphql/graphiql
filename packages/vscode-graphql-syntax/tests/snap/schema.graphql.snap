>type Query {
#^^^^ source.graphql meta.type.interface.graphql keyword.type.graphql
#    ^ source.graphql meta.type.interface.graphql
#     ^^^^^ source.graphql meta.type.interface.graphql support.type.graphql
#          ^ source.graphql meta.type.interface.graphql meta.type.object.graphql
#           ^ source.graphql meta.type.interface.graphql meta.type.object.graphql punctuation.operation.graphql
>  field: String
#^^ source.graphql meta.type.interface.graphql meta.type.object.graphql
#  ^^^^^ source.graphql meta.type.interface.graphql meta.type.object.graphql variable.graphql
#       ^ source.graphql meta.type.interface.graphql meta.type.object.graphql punctuation.colon.graphql
#        ^ source.graphql meta.type.interface.graphql meta.type.object.graphql
#         ^^^^^^ source.graphql meta.type.interface.graphql meta.type.object.graphql support.type.builtin.graphql
>}
#^ source.graphql meta.type.interface.graphql meta.type.object.graphql punctuation.operation.graphql
>
>type Mutation {
#^^^^ source.graphql meta.type.interface.graphql keyword.type.graphql
#    ^ source.graphql meta.type.interface.graphql
#     ^^^^^^^^ source.graphql meta.type.interface.graphql support.type.graphql
#             ^ source.graphql meta.type.interface.graphql meta.type.object.graphql
#              ^ source.graphql meta.type.interface.graphql meta.type.object.graphql punctuation.operation.graphql
>  doTheThing(input: InputType!): String
#^^ source.graphql meta.type.interface.graphql meta.type.object.graphql
#  ^^^^^^^^^^ source.graphql meta.type.interface.graphql meta.type.object.graphql variable.graphql
#            ^ source.graphql meta.type.interface.graphql meta.type.object.graphql meta.brace.round.graphql
#             ^^^^^ source.graphql meta.type.interface.graphql meta.type.object.graphql meta.variables.graphql variable.parameter.graphql
#                  ^ source.graphql meta.type.interface.graphql meta.type.object.graphql meta.variables.graphql punctuation.colon.graphql
#                   ^ source.graphql meta.type.interface.graphql meta.type.object.graphql meta.variables.graphql
#                    ^^^^^^^^^ source.graphql meta.type.interface.graphql meta.type.object.graphql meta.variables.graphql support.type.graphql
#                             ^ source.graphql meta.type.interface.graphql meta.type.object.graphql meta.variables.graphql keyword.operator.nulltype.graphql
#                              ^ source.graphql meta.type.interface.graphql meta.type.object.graphql meta.brace.round.graphql
#                               ^ source.graphql meta.type.interface.graphql meta.type.object.graphql punctuation.colon.graphql
#                                ^ source.graphql meta.type.interface.graphql meta.type.object.graphql
#                                 ^^^^^^ source.graphql meta.type.interface.graphql meta.type.object.graphql support.type.builtin.graphql
>}
#^ source.graphql meta.type.interface.graphql meta.type.object.graphql punctuation.operation.graphql
>
>input InputType {
#^^^^^ source.graphql meta.type.interface.graphql keyword.input.graphql
#     ^ source.graphql meta.type.interface.graphql
#      ^^^^^^^^^ source.graphql meta.type.interface.graphql support.type.graphql
#               ^ source.graphql meta.type.interface.graphql meta.type.object.graphql
#                ^ source.graphql meta.type.interface.graphql meta.type.object.graphql punctuation.operation.graphql
>  field: String
#^^ source.graphql meta.type.interface.graphql meta.type.object.graphql
#  ^^^^^ source.graphql meta.type.interface.graphql meta.type.object.graphql variable.graphql
#       ^ source.graphql meta.type.interface.graphql meta.type.object.graphql punctuation.colon.graphql
#        ^ source.graphql meta.type.interface.graphql meta.type.object.graphql
#         ^^^^^^ source.graphql meta.type.interface.graphql meta.type.object.graphql support.type.builtin.graphql
>}
#^ source.graphql meta.type.interface.graphql meta.type.object.graphql punctuation.operation.graphql
>